TombEngine/Game/itemdata/creature_info.h

```diff
@@ -71,6 +71,60 @@ struct LOTInfo
 	bool CanMonkey	  = false;
 };
 
+struct CreatureBiteInfo
+{
+	Vector3i Position = Vector3i::Zero;
+	int BoneID = -1;
```

Comment:
We have `NO_JOINT` constant which can be utilized here instead of magic number.

Reply:
Done.